esp/bindings/http/platform/httptransport.cpp

```diff
@@ -798,6 +798,44 @@ void CHttpMessage::logSOAPMessage(const char* message, const char* prefix)
     return;
 }
 
+static bool skipLogContent(const char* httpHeader)
+{
+    VStringBuffer postStr("%s ", POST_METHOD);
+    if (!startsWith(httpHeader, postStr.str()))
+        return false;
+
+    const char* servicePtr = httpHeader + 5;
+    if (isEmptyString(servicePtr) || (servicePtr[0] != '/'))
+        return false;
+
+    const char* methodPtr = strchr(++servicePtr, '/');
+    if (!methodPtr)
+        return false;
+
+    StringBuffer espService;
```

Comment:
Since we already know it's a service we care about we may not need the StringBuffer for espService at all?